app/assets/javascripts/discourse/app/components/hamburger-menu-wrapper.js

```diff
@@ -0,0 +1,7 @@
+import Component from "@ember/component";
+import layout from "discourse/templates/components/hamburger-menu-wrapper";
+
+export default Component.extend({
+  layout,
+  tagName: "",
+});
```

Comment:
I think we can delete this component now (and its template below).

Reply:
O wait I thought I've already deleted it :thinking: Must have missed it in my commit.

Reply:
Good catch I've resolved it.